tests/test_to_process.py

```diff
@@ -48,6 +48,7 @@ def process_func(receiver: int, sender: int) -> None:
     os.write(sender, data + b", World!")
 
 
+@pytest.mark.parametrize("anyio_backend", ["asyncio"])
```

Comment:
Wait, why it this only tested with asyncio? I'm not willing to support any feature that isn't available across the different back-ends.

Reply:
I'm not sure if the test was hanging for other event loops or because of the platform, I'll try to remove this line.

Reply:
So after https://github.com/agronholm/anyio/pull/993/commits/7c3f196eef39229ac1e0b54c1d81304be4def918 it seems that the test hangs with `asyncio+uvloop`.

Reply:
And I'm not sure why the test hangs on Windows. [Pipes](https://docs.python.org/3/library/os.html#os.pipe) are supposed to work, and [inheritance of file descriptors](https://docs.python.org/3/library/os.html#inheritance-of-file-descriptors) should be supported.

Reply:
This will nonetheless have to be resolved before I will consider merging this.

Reply:
So the issue is only with `uvloop` on macOS (bad file descriptor). Not sure what to do.